A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -379,6 +389,18 @@ var A320_Neo_UpperECAM;
                         ]
                     },
                     //Airborne
+                    {
+                        name: "OVERSPEED",
+                        messages: [
+                            {
+                                message: "",
+                                level: 3,
+                                isActive: () => {
+                                    return this.getLimitSpeed();
+                                },
```

Comment:
Pretty sure you can just do this here.

```suggestion
                                isActive: this.getLimitSpeed,
```

Reply:
Could but I think it looks cleaner in my opinion

Reply:
Could but probs best to stick with the style currently of the rest of the file

Reply:
True